kettle-columnstore-bulk-exporter-plugin/src/main/java/com/mariadb/columnstore/api/kettle/KettleColumnStoreBulkExporterStepDialog.java

```diff
@@ -679,7 +683,17 @@ private void updateTableView(){
           }
         }
       } catch (ColumnStoreException e) {
-        logDetailed("Can't access the ColumnStore table " + wTargetDatabaseFieldName.getText() + " " + wTargetTableFieldName.getText());
+        if(e.getMessage().toLowerCase().contains("connection failure")){
+          logBasic("Can't connect to ColumnStore server: " + e.getMessage());
+          columnStoreReachable = false;
```

Comment:
Why can't we just error out error rather than have all the conditional changes later?